src/flask_allows/requirements.py

```diff
@@ -1,9 +1,11 @@
 import operator
 from abc import ABCMeta, abstractmethod
 from functools import wraps
+from inspect import isclass
+from types import FunctionType
 
 from flask import request
-from flask._compat import with_metaclass
```

Comment:
Since Python 3 supports ``metaclass``, wouldn't it be better to use this instead?

Reply:
Good catch.  I was trying to make the minimum number of changes necessary to support Flask 2, but this more straight forward and reflected in the most recent commit.